hera_cal/io.py

```diff
@@ -380,7 +380,7 @@ def load_cal(input_cal, return_meta=False):
 
     Returns:
         if return_meta is True:
-            (gains, flags, quals, total_qual, ants, freqs, times, pols)
+            (gains, flags, quals, total_qual, ants, freqs, times, pols, history)
```

Comment:
Is now the time to switch to returning gains, flags, and a meta dictionary? That would be a more extensible format. Alternatively, we can hold off for now and make it an issue so we can get moving on the pipeline. Thoughts?

Reply:
yeah let's just make that its own PR seeing as its somewhat of a separate issue